Client.php

```diff
@@ -67,6 +67,37 @@ public function createEngine($name, $language = null)
         return $this->performRequest($endpoint);
     }
 
+    /**
+     * Create a new API key.
+     *
+     * Documentation: https://swiftype.com/documentation/app-search/api/credentials#create
+     *
+     * @param string $name             API key name.
+     * @param string $type             Type of API key.
+     * @param bool   $read
+     * @param bool   $write
+     * @param bool   $accessAllEngines
+     * @param array  $engines
+     *
+     * @return array
+     */
+    public function createKey($name, $type, $read = null, $write = null, $accessAllEngines = null, $engines = null)
```

Comment:
`null` default values does not seems a great choice for boolean.
Can you test using a boolean in the spec.